scipy/fft/_basic.py

```diff
@@ -151,6 +151,7 @@ def fft(x, n=None, axis=-1, norm=None, overwrite_x=False, workers=None, *,
     In this example, real input has an FFT which is Hermitian, i.e., symmetric
     in the real part and anti-symmetric in the imaginary part:
 
+    >>> import numpy as np
```

Comment:
Remove this line.  The import was already done a few lines earlier.

Reply:
I have committed the new code for changes as per the reviews.

Reply:
The redundant import is still there.
```suggestion
```